api/lib/source/cosmosV0-source.js

```diff
@@ -361,10 +361,7 @@ class CosmosV0API extends RESTDataSource {
     // if we don't add the staking denom, we would show a 0 total for the staking denom which is wrong
     if (
       delegations.length > 0 &&
-      !balancesResponse.find(
-        ({ denom }) =>
-          this.reducers.denomLookup(denom) === this.network.stakingDenom
-      )
+      !balancesResponse.find(({ denom }) => denom === this.network.stakingDenom)
```

Comment:
? this would fail right? the balance response has the balances not yet in Lunie format